tests/scene/test_font.h

```diff
@@ -0,0 +1,79 @@
+#ifndef TEST_FONT_H
+#define TEST_FONT_H
+
+#include "scene/resources/font.h"
+#include "tests/test_macros.h"
+#include "tests/test_utils.h"
+
+namespace TestFont {
+TEST_CASE("[String] Instantiation") {
+	Ref<FontFile> ff;
+	ff.instantiate();
+
+	REQUIRE_MESSAGE(ff != nullptr, "Reference should not be null");
+}
+
+// Macro to generate tests for getters
+#define GETTER_TEST(name, getter, default_value)                                                                                      \
```

Comment:
```suggestion
#define GETTER_TEST(m_name, m_getter, m_default_value)                                                                                \
```
Prefix macro args with `m_`